### ISSUE TITLE
Assert failure: _safePointDecoder.IsSafePoint(curOffs)

### ISSUE THREAD

**Issue by VincentBu** [dotnet/coreclr#25048]
**Job:**
https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190608.1/workItem/baseservices.threading/analysis/xunit/baseservices_threading._interlocked_compareexchange_compareexchangeobject_compareexchangeobject_~2F_interlocked_compareexchange_compareexchangeobject_compareexchangeobject_sh

**Failed tests**
baseservices_threading._interlocked_compareexchange_compareexchangeobject_compareexchangeobject_._interlocked_compareexchange_compareexchangeobject_compareexchangeobject_sh

**Log:**
```
Assert failure(PID 1564 [0x0000061c], Thread: 1978 [0x07ba]): _safePointDecoder.IsSafePoint(curOffs)
    File: /__w/4/s/src/vm/eetwain.cpp Line: 5015
    Image: /root/helix/work/correlation/corerun

/root/helix/work/workitem/baseservices/threading/interlocked/compareexchange/compareexchangeobject/compareexchangeobject.sh: line 252:  1564 Aborted                 (core dumped) $LAUNCHER $ExePath \"${CLRTestExecutionArguments[@]}\"

Return code:      1
Raw output file:      /root/helix/work/workitem/baseservices/threading/Reports/baseservices.threading/interlocked/compareexchange/compareexchangeobject/compareexchangeobject.output.txt
Raw output:
BEGIN EXECUTION
/root/helix/work/correlation/corerun compareexchangeobject.exe ''
Expected: 100
Actual: 134
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/root/helix/work/correlation
> /root/helix/work/workitem/baseservices/threading/interlocked/compareexchange/compareexchangeobject/compareexchangeobject.sh
Expected: True
Actual:   False
```

**Comment by VincentBu** [comment]
Also seen in:
https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190608.1/workItem/JIT.Directed/analysis/xunit/JIT_Directed._Misc_function_pointer_MutualThdRecur_fptr_MutualThdRecur_fptr_~2F_Misc_function_pointer_MutualThdRecur_fptr_MutualThdRecur_fptr_sh

**Comment by VincentBu** [comment]
some tests failed  due to the assertion:
[baseservices_threading._interlocked_compareexchange_CompareExchangeLong_1_CompareExchangeLong_1_._interlocked_compareexchange_CompareExchangeLong_1_CompareExchangeLong_1_sh](https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190609.1/workItem/baseservices.threading/analysis/xunit/baseservices_threading._interlocked_compareexchange_CompareExchangeLong_1_CompareExchangeLong_1_~2F_interlocked_compareexchange_CompareExchangeLong_1_CompareExchangeLong_1_sh)

[baseservices_threading._generics_Monitor_EnterExit09_EnterExit09_._generics_Monitor_EnterExit09_EnterExit09_sh](https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190609.1/workItem/baseservices.threading.generics/analysis/xunit/baseservices_threading._generics_Monitor_EnterExit09_EnterExit09_~2F_generics_Monitor_EnterExit09_EnterExit09_sh)

[Interop_ICustomMarshaler._Primitives_ICustomMaraler_ICustomMaraler_._Primitives_ICustomMarshaler_ICustomMarshaler_sh](https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190609.1/workItem/Interop/analysis/xunit/Interop_ICustomMarshaler._Primitives_ICustomMaraler_ICustomMaraler_~2F_Primitives_ICustomMarshaler_ICustomMarshaler_sh)

[JIT_Performance._CodeQuality_BenchmarksGame_mandelbrot_mandelbrot_7_mandelbrot_7_._CodeQuality_BenchmarksGame_mandelbrot_mandelbrot_7_mandelbrot_7_sh](https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190609.1/workItem/JIT/analysis/xunit/JIT_Performance._CodeQuality_BenchmarksGame_mandelbrot_mandelbrot_7_mandelbrot_7_~2F_CodeQuality_BenchmarksGame_mandelbrot_mandelbrot_7_mandelbrot_7_sh)

more in:
https://mc.dot.net/#/user/coreclr-outerloop-gcstress-extra/ci~2Fdotnet~2Fcoreclr~2Frefs~2Fheads~2Fmaster/test~2Ffunctional~2Fcli~2F/20190609.1/workItem/JIT.Directed/analysis/xunit/JIT_Directed._Misc_function_pointer_MutualThdRecur_fptr_MutualThdRecur_fptr_~2F_Misc_function_pointer_MutualThdRecur_fptr_MutualThdRecur_fptr_sh

**Comment by briansull** [comment]
Likely that this issue is fixed, but still waiting on the testing infrastructure to confirm this

**Comment by briansull** [comment]
Resolved by #25445

**Comment by briansull** [comment]
No new occurrences according to my Kusto queury
